### DISCUSSION TITLE
keepassxc.ini config file - can this be stored publicly?

### DISCUSSION THREAD

**Discussion by charger89** [keepassxreboot/keepassxc#10055] [Q&A] [answered]
I'd like to store keepassxc.ini in a public git repository alongside other dotfiles. It appears to contain some key in it, assuming it's to do with browser integration. Does this mean the file essentially contains secrets and should be kept private?

**Comment by phoerious** [comment, marked as answer]
The keys are for KeeShare, but we don't use them anymore. You can simply delete them. The main config ini file contains no sensitive information.

**Reply by charger89** [reply]
Sweet! May the whole `[KeeShare]` block be nuked from config, or only the `Active, Foreign, Own` items under it?

**Reply by phoerious** [reply]
Can delete anything. Worst case it will be recreated automatically.

**Reply by e-t-l** [reply]
I was about to ask the same question! 

> keys are for KeeShare, but we don't use them anymore.

Could you very briefly elaborate, what is/was keeshare and why is it deprecated?

**Reply by droidmonkey** [reply]
Read our userguide for that information. Keeshare is not deprecated just the signing aspect. It really isn't relevant.
